pulp_rpm/app/models/repository.py

```diff
@@ -272,6 +276,12 @@ def artifacts_for_version(version):
 
     class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
+        permissions = [
+            ("sync_rpmrepository", "Sync a repository"),
+            ("copy_rpmrepository", "Copy a repository"),
```

Comment:
is it correct to define permissions on the model from a different viewset?

Reply:
In case of `copy_rpmrepository`, `copy` doesn't have own model and it is not binded to repository endpoint (can't remember the reason why) directly but imo it is still repository action, so it looks like the best place to define it on repository model.

Reply:
I agree. And i think we cannot assume a 1-1 relationship between models and viewsets. So this copy is still an action performed by a (specialized) repository view.